pkg/compute/compute_integration_test.go

```diff
@@ -255,7 +255,10 @@ func TestInit(t *testing.T) {
 				CreateDomainFn:  createDomainOK,
 				CreateBackendFn: createBackendOK,
 			},
-			manifestIncludes: `authors = ["test@example.com"]`,
+			manifestIncludes: []string{
+				`authors = ["test@example.com"]`,
+				`name = "fastly-build"`,
```

Comment:
As this is an `AssertStringContains` string contains match and we're only matching on the prefix of the name, you'll need to drop the closing `"` here for this to pass.

Reply:
Yup. I noticed after I pushed it. Anyways, I've reverting that change and add another test as you originally suggested.